e2e/chat.spec.ts

```diff
@@ -694,4 +694,58 @@ describe('chat', () => {
 
     await expect(page.getByText('Start a new chat to begin')).toBeVisible();
   });
+
+  test('allows user to upload multiple supported files', async ({ page }) => {
+    const maxFileUploads = 4;
+    test.setTimeout(30 * 1000);
+
+    const chat = new Chat(page);
+    await chat.goto();
+
+    const buffer = Buffer.from(
+      'iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAYAAAAfFcSJAAAADUlEQVR42mP8z8BQDwAEhQGAhKmMIQAAAABJRU5ErkJggg==',
+      'base64',
+    );
+
+    const withinLimitSupportedFiles = [...Array(maxFileUploads)].map(() => ({
+      name: 'test-image.png',
+      mimeType: 'image/png',
+      buffer,
+    }));
+
+    const fileInput = page.locator('input[type="file"]');
+    await fileInput.setInputFiles(withinLimitSupportedFiles);
```

Comment:
I had this working locally by clicking the icon and then awaiting the filechooser, but that didn't work in CI/CD